docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Troubleshooting"
+description: |
+  Troubleshooting source integrations.
+---
+
+## Unable to clone a repository
+
+If you [add](/administration/users.md#add-a-user-to-a-project) a user with **Project Admin** role to a project on Platform.sh, but you haven’t added them to the remote repository on GitHub, GitLab, or Bitbucket, they won't be able to clone the project locally.
+
+If the user you added to a project is using `platform get` with the CLI:
+```
+$ platform get <projectID>
+```
+it would return the following error:
+
+```
+Failed to connect to the Git repository: git@github.com:user/github-repo.git
+Please make sure you have the correct access rights and the repository exists.
+```
+
+### Check the external repository access rights (recommended solution)
+
+Ensure the user has the correct access rights for the external integration repository.
+
+### Clone Platform.sh repository
+
+Clone the repository from Platform.sh, which offers a read-only mirror repository of the external integration repository.
+
+```bash
+$ git clone <PROJECT_ID>@git.<REGION>.platform.sh:<PROJECT_ID>.git
```

Comment:
The `git clone` command offered in the console includes the project name, so it makes sense to me to include it here.

On further thought, it might also make sense to mention they can find the command in the console, as otherwise the variables there might not be clear to everyone.